src/Employer/Employer.Web/Views/VacancyPreview/VacancyPreview.cshtml

```diff
@@ -36,9 +36,16 @@
 }
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState, OrderedFieldNames = Model.OrderedFieldNames }' />
+        <div asp-show="@Model.IsTransferred">
```

Comment:
So this bit of view is being repeated in ManageVacancy.cshtml. This is where a partial view with any alerts could help... again still thinking out loud on my previously mentioned idea.

Reply:
Will keep in mind.